test/Intl/rlexe.xml

```diff
@@ -11,28 +11,28 @@
   <test>
     <default>
       <files>Collator.js</files>
-      <tags>Intl,require_winglob</tags>
+      <tags>Intl</tags>
     </default>
   </test>
   <test>
     <default>
       <files>CollatorOptions.js</files>
       <compile-flags>-args summary -endargs</compile-flags>
-      <tags>Intl,require_winglob</tags>
+      <tags>Intl</tags>
     </default>
   </test>
   <test>
     <default>
       <files>DateTimeFormatInvalidOptions.js</files>
       <compile-flags>-args summary -endargs</compile-flags>
-      <tags>Intl</tags>
+      <tags>Intl,require_winglob</tags>
```

Comment:
>require_winglob [](start = 17, length = 15)

interesting... plans to fix this?

Reply:
Yes, this is related to hour12 being broken.

Reply:
Okay, add a comment to the effect of when this test should be re-enabled.

Reply:
the PR isnt getting merged until this test is re-enabled for ICU so I wouldn't worry about it too much.